tox.ini

```diff
@@ -28,43 +28,43 @@ deps =
 
 
 # Earliest supported Scrapy version.
-[testenv:pinned-scrapy-2.0]
```

Comment:
The new update on `tox >= 4.0` had a lot of backward incompatible changes.

It's now strict on the `testenv:<env_name>` format which renders this old line to have this error:

```
pinned-26-scrapy: failed with env name pinned-scrapy-2.0 conflicting with base python python3.7
pinned-scrapy-2.0: FAIL code 1 (0.01 seconds)
evaluation failed :( (1.24 seconds)
```

Setting `ignore_basepython_conflict=True` is supposed to fix this but it doesn't work. Seems like they're still releasing a lot of regression fixes after the `4.0` release.

They're also quite aggressive on the name inference. If they see any two digits in the env name, they'd treat it as a Python version. So other workarounds like `20`, `2-0`, etc doesn't work.